lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -2049,7 +2050,7 @@ module.exports = (config) => {
 
       const sessionInfo = {
         delta: cfg.delta,
-        deviceTags: ['insulin-pump'],
+        deviceTags: ['insulin-pump', 'cgm'],
```

Comment:
For time-check to work correctly, you need to define the `deviceTags` at the beginning just after you've cloned `cfg`, as `cfg.deviceTags`, and then pass them to `sessionInfo` here as `deviceTags: cfg.deviceTags,`. This is because we need to know the type of the device when we check the time in order to display the correct message.